docs/src/main/asciidoc/README.adoc

```diff
@@ -1,5 +1,5 @@
-include::./attributes.adoc[]
 = How to Create {project-name} Documentation
```

Comment:
We should replace the variable by `Quarkus`

Reply:
Ok if not too many files impacted I'll try to remove attribute project-name and hardcode Quarkus where it is used. I imagine it was only relevant while the project name wasn't known/final yet.

Reply:
Yeah, exactly. We can remove it now.

But I would fix these two in this commit and do the big replace in a separate commit.